SU-SOE/soe_basic
# D8CORE-2282: adding small fix for Engineering Lockup on H, I, and P

# READY FOR REVIEW

# Summary
- Fixing the Engineering line breaking at the G in the Lockup for option H, I, and P.

# Review By (Date)
- 11/13

# Urgency
- High

# Steps to Test

1.Pull in this change.
2. Notice the lockup in the header and the hanging G.
3. Update the lock up to use H, I and P and verify the G is no longer hanging below Engineering.
4. Verify no changes to the footer. 
5. Verify the responsive version looks correct.


# Affected Projects or Products
- SOE Lockup

# Associated Issues and/or People
- D8CORE-2282

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
